### ISSUE TITLE
ConnectionAbortedException ---> Reading is already in progress

### ISSUE THREAD

**Issue by sepehr1014** [aspnet/AspNetCore#17062]
### Describe the bug
We're frequently getting the exception below as soon as we migrated from 2.2.7 to 3.0.

```
Microsoft.AspNetCore.Connections.ConnectionAbortedException: The connection was aborted by the application.
 ---> System.InvalidOperationException: Reading is already in progress.
   at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Http.Http1ContentLengthMessageBody.TryReadInternal(ReadResult& readResult)
   at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Http.Http1MessageBody.OnConsumeAsync()
   --- End of inner exception stack trace ---
```

### Further technical details
- ASP.NET Core 3.0
- IIS on Windows Server 2016
- Using OutOfProcess hosting model

**Comment by Tratcher** [comment]
Possible duplicate of #14727 @halter73 ?

**Comment by sepehr1014** [comment]
BTW we only have APIs, and they are only used by our own app (using Volley on Android). So the requests are not coming from web browsers.

**Comment by halter73** [comment]
Thanks for the report. I agree this is a duplicate of #14727. There should be no functional impact to this issue, but the first chance exception and log warning are definitely not good. This should be fixed in 3.1.
